model/enduser/registry.yaml

```
@@ -0,0 +1,18 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes information about the end user, which can be used as a subdomain of browser, client, or user domains.
```

```suggestion
      Describes the end user.
```


I don't think the other part is correct:

> which can be used as a subdomain of browser, client, or user domains.